### PR TITLE
Drawing to use new notification for ChangeHistory

### PR DESCRIPTION
**Pull request by pllim**
Follow up of #621.

### CODE REVIEW

File: ginga/rv/plugins/ChangeHistory.py [hunk 1]
```diff
@@ -18,26 +18,24 @@
...
-        # This issues the 'modified' callback and sets the timestamp
+        # This changes the data buffer
         image.set_data(new_data, ...)
-        # Manually add the description
-        chname = self.fv.get_channel_name(self.fitsimage)
-        channel = self.fv.get_channel_info(chname)
-        iminfo = channel.get_image_info(image.get('name'))
-        iminfo.reason_modified = 'Something was done to this image buffer'
+        # Add description for ChangeHistory
+        info = dict(time_modified = datetime.utcnow(),
```
Comment (ejeschke) [inline ginga/rv/plugins/ChangeHistory.py]: I think the spaces around the "=" will trigger the flake8 failure.

Comment (pllim) [reply ginga/rv/plugins/ChangeHistory.py]: That is just the docstring, but yes, we should give people a good example. I'll fix it when I get the chance. Thanks!

Comment (ejeschke) [reply ginga/rv/plugins/ChangeHistory.py]: No, in the dict(), right?

Comment (pllim) [reply ginga/rv/plugins/ChangeHistory.py]: This is a code snippet *inside* a docstring, that is why `flake8` check is not catching it, but I agree it should be fixed.

Comment (ejeschke) [reply ginga/rv/plugins/ChangeHistory.py]: Sorry, I guess I'm not looking at the full code, just the patch...

**Comment by pllim** [comment]
@ejeschke , it is done. Thanks for the review!

**Comment by ejeschke** [comment]
Ok, merging.

**Comment by pllim** [comment]
Thanks!

**Pull request merged**